echopype/echodata/combine.py

```diff
@@ -181,12 +181,12 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
                     combined_group["transceiver_software_version"] = combined_group[
                         "transceiver_software_version"
                     ].astype("<U10")
-                    combined_group["channel_id"] = combined_group["channel_id"].astype("<U50")
+                    combined_group["channel"] = combined_group["channel"].astype("<U50")
                 elif sonar_model == "EK60":
                     combined_group["gpt_software_version"] = combined_group[
                         "gpt_software_version"
                     ].astype("<U10")
-                    combined_group["channel_id"] = combined_group["channel_id"].astype("<U50")
+                    combined_group["channel"] = combined_group["channel"].astype("<U50")
```

Comment:
this `.astype("<U50")`, I seem to remember that it was necessary to make encoding work. @emiliom do you remember if this is true?

Reply:
No, I don't remember being involved in this

Reply:
Ok, I will put a TODO that will address this later.

Reply:
```suggestion
                    # TODO: investigate further why we need to do .astype("<U50")
                    combined_group["channel"] = combined_group["channel"].astype("<U50")
```